modules/pricer/src/main/java/com/opengamma/strata/pricer/swaption/VolatilitySwaptionPhysicalProductPricer.java

```diff
@@ -89,7 +89,7 @@ public CurrencyAmount presentValue(
     if (expiry < 0d) { // Option has expired already
       return CurrencyAmount.of(fixedLeg.getCurrency(), 0d);
     }
-    double forward = swapPricer.parRate(underlying, ratesProvider);
+    double forward = forwardRate(swaption, ratesProvider);
```

Comment:
Similar to how you have changed this to the `forwardRate()` method, you can do the same for the `impliedVolatility()` method. Probably worth doing for consistency if we are doing the same for the forward rates across the other swaption pricers.

Reply:
This is already done.